lib/alerts_viewer_web/controllers/bus_controller.ex

```diff
@@ -9,7 +9,7 @@ defmodule AlertsViewerWeb.BusController do
     mod = String.to_existing_atom(algorithm)
     bus_routes = Routes.all_bus_routes()
     stats_by_route = RouteStatsPubSub.all()
-    bus_alerts = Alerts.all()
+    bus_alerts = Alerts.all() |> Alerts.by_service("3") |> Alerts.by_effect("delay")
```

Comment:
💡 I didn't think of this one. But is it already covered by the `delay_alert?` check on the following line?

Reply:
Oh, yeah, unlike the other file that is the only place that it's used